### ISSUE TITLE
Latest Stable Version Cause CTD at Large Airports with latest driver (RTX 3090)

### ISSUE THREAD

**Issue by runningdavid** [flybywiresim/a32nx#5562]
### Aircraft Version

Stable

### Build info

```json
{
  "built": "2021-08-04T00:57:39+00:00",
  "ref": "v0.6.2",
  "sha": "1f940ca300187689e4f86881744cffc1afecf4d7",
  "actor": "2hwk",
  "event_name": "manual"
}
```

### Describe the bug

Just got my RTX 3090 yesterday and updated driver to latest. I'm able to loading into large airports (e.g. LAX) with default planes no problem. But with A32NX it crashes very often. Tried various liveries (house, FBW default, etc.) and still crashes. It's especially bad at LAX where it crashes every single time. I first heard the radioaltimeter sound (10) upon spawning onto the runway and 3 seconds later everything froze and sim went CTD.

On medium/small airports it works fine. Probably because LAX is a large airport and the city close by is huge, which put a lot stress on the GPU. But it works with default aircrafts so I'm suspecting maybe there's something to do with the A32nx.

Also I didn't have crash problem with A32NX dev version on my older GTX 1080 card (even though FPS is not great). But once I switched to RTX 3090 and updated drivers I immediately encountered several crashes at several airports (large or small). So I switched to stable now it fixed the crash on small and medium airports but still not large airports.

### Expected behavior

Sim should not CTD when loading A32NX onto LAX runway 7R, running on RTX 3090 and latest Nvidia driver.

### Steps to reproduce

Load A32NX to LAX runway 7R, with a machine running RTX 3090 and latest NVidia driver.

### Additional info (optional)

I love this plane and appreciate the work all of you did to constantly improve it.

**Comment by NUTTTTTTTT** [comment]
Please read the release notes above, and check reported issues before opening a new issue on Github. 

![image](https://user-images.githubusercontent.com/87286435/128550412-49b55ff1-5293-4c4a-a49b-b21dcd8b3fc5.png)

**Comment by runningdavid** [comment]
Ok didn't know there were deeper issues. Thanks!

**Comment by bouveng** [comment]
Also, with regards to CTD's.

Official CTD-guide.
https://flightsimulator.zendesk.com/hc/en-us/articles/360015893879-Crashing-CTDs-issues

**Comment by runningdavid** [comment]
Maybe my GPU is too new. Old GTX 1080 doesn't crash.

**Comment by sidnov** [comment]
There is another fix being worked on. Please stay tuned for a release announcement..

**Comment by runningdavid** [comment]
I can confirm CTD issues has been after PR checkin https://github.com/flybywiresim/a32nx/pull/5564

**Comment by TLSY2** [comment]
I think that the ctd issues are simple, its because not a lot of ram usage and when loading its 100% cpu use, windows automatically shuts down the app.

**Comment by runningdavid** [comment]
> I think that the ctd issues are simple, its because not a lot of ram usage and when loading its 100% cpu use, windows automatically shuts down the app.

Not the case for me. My case it was a bug and was fixed with PR #5564

**Comment by MisterChocker** [comment]
Closing since issue resolved with #5564.

If the issue appears again feel free to re-open.
